ammonia-CO-H2-Alzueta-2023.yaml

```diff
@@ -0,0 +1,2549 @@
+  - {P: 1.0 atm, A: 1.4e+23, b: -2.355, Ea: 1.7519e+04}
+  - {P: 10.0 atm, A: 7.3e+23, b: -2.665, Ea: 1.7634e+04}
+  note: |-
+     Troe J JPCA 109:8320-8328 2005
+     J. Troe, J. Phys. Chem. A 111 (2007) 38623867.
+    CH2O+H=HCO+H2                        5.9E03   3.130    1510 !
+     Wang SK Dames EE Davidson DF Hanson RK, J. Phys. Chem. A  118 10201-10209 2014
+     applying prompt HCO dissociation - Labbe et al., J. Phys. Chem. Lett. 2016:
+     1 atm
+- equation: CH2O + H <=> HCO + H2
+  duplicate: true
+  type: pressure-dependent-Arrhenius
+  rate-constants:
+  - {P: 0.04 atm, A: 2.1e+10, b: 1.057, Ea: 3720.0}
+  - {P: 1.0 atm, A: 1.6e+15, b: -0.444, Ea: 5682.0}
+  - {P: 10.0 atm, A: 4.2e+09, b: 1.294, Ea: 3591.0}
+  note: 1 atm
+- equation: CH2O + H <=> H + CO + H2
+  type: pressure-dependent-Arrhenius
+  rate-constants:
+  - {P: 0.04 atm, A: 7.2e+08, b: 1.903, Ea: 1.1733e+04}
+  - {P: 1.0 atm, A: 5.1e+07, b: 2.182, Ea: 1.1524e+04}
+  - {P: 10.0 atm, A: 1.1e+09, b: 1.812, Ea: 1.3163e+04}
+- equation: CH2O + O <=> HCO + OH
+  duplicate: true
+  rate-constant: {A: 5.6e+31, b: -5.189, Ea: 1.9968e+04}
+  note: |-
+    CH2O+O=HCO+OH                        4.2E11   0.570    2760 !
+     Baulch DL Bowman CT Cobos CJ Cox RA Just Th Kerr JA Pilling MJ Stocker D Troe J Tsang W Walker RW Warnatz J JPCRD 34:757-1397 2005
+     applying prompt HCO dissociation - Labbe et al., J. Phys. Chem. Lett. 2016:
+- equation: CH2O + O <=> HCO + OH
+  duplicate: true
+  rate-constant: {A: 1.4e+15, b: -0.53, Ea: 4011.0}
+- equation: CH2O + O <=> H + CO + OH
+  rate-constant: {A: 2.5e+21, b: -1.903, Ea: 2.2674e+04}
+- equation: CH2O + OH <=> HCO + H2O
+  rate-constant: {A: 1.9e+09, b: 1.256, Ea: -302.0}
+  note: |-
+    CH2O+OH=HCO+H2O                      2.4E07   1.830   -1116 !
+     S. Wang, D. F. Davidson, R. K. Hanson, Proc. Combust. Inst. 35 473-480 2015
+     applying prompt HCO dissociation - Labbe et al., J. Phys. Chem. Lett. 2016:
+- equation: CH2O + OH <=> H + CO + H2O
+  rate-constant: {A: 7.2e+10, b: 0.892, Ea: 9310.0}
+  note: |-
+    PLOG/0.04                           3.6E09    1.167    -206/  !
+    PLOG/1	                             1.9E09    1.256    -302/  !
+    PLOG/10                             1.1E09    1.330    -392/  !
+- equation: CH2O + HO2 <=> HCO + H2O2
+  rate-constant: {A: 4.4e+08, b: 1.298, Ea: 1.2129e+04}
+  note: |-
+     PLOG/0.04                           7.0E10    0.911    8646/
+     PLOG/1                              7.2E10    0.892    9310/
+     PLOG/10                             8.4E10    0.879    9843/
+    CH2O+HO2=HCO+H2O2                    4.1E04   2.500   10206 !
+     Eiteneer B Yu CL Goldenberg M Frenklach M JPCA 102:5196 1998
+     applying prompt HCO dissociation - Labbe et al., J. Phys. Chem. Lett. 2016:
+    CH2O+HO2=HCO+H2O2                    4.1E04   2.500   10206 !
+     DUPLICATE
+    CH2O+HO2=HCO+H2O2                   -2.5E14   0.027   30133
+     DUPLICATE
+     refit
```

Comment:
There are a number of cases like this, too, where there are commented-out reactions and their corresponding references getting attached to some other reaction.

You might find it easiest to clean these up by creating a version of the Chemkin input file with all the commented-out reactions and rate terms simply removed, changing the order of the comments with respect to their reactions, and then reconverting the file.